app/core/api/levels.coffee

```diff
@@ -1,4 +1,6 @@
 fetchJson = require './fetch-json'
 
 module.exports = {
+  getByOriginal: (original) ->
+    return fetchJson("/db/level?original=#{original}")
```

Comment:
Don't need a new endpoint, instead do `/db/level/#{original}/version`.

Reply:
Ah, couldn't find it. Might be a good candidate for a bus ride refactor ;)

(I also didn't realize server/routes/db.coffee existed. What exactly does it handle?)